lib/src/main/scala/spinal/lib/bus/localbus/sim/MemBusDriver.scala

```diff
@@ -0,0 +1,43 @@
+package spinal.lib.bus.localbus
+
+import spinal.core._
+import spinal.core.sim._
+
+case class MemBusDriver(bus : MemBus, clockdomain : ClockDomain) {
+  var verbose = false
+  def write(address : BigInt, data : BigInt) : Unit = {
+    if(verbose) println(s"Mem[0x${address.toString(16)}] = 0x${data.toString(16)}")
+    bus.ce   #= true
+    bus.wr   #= true
+    bus.addr #= address
+    bus.wdat #= data
+    clockdomain.waitSampling()
+    bus.ce   #= false
+    bus.wr.randomize()
+    bus.addr.randomize()
+    bus.wdat.randomize()
+  }
+
+  def read(address : BigInt) : BigInt = {
+    bus.ce #= true
+    bus.wr #= false
+    bus.addr #= address
+    bus.wdat #= 1234 //.randomize()
```

Comment:
this should use randomize like in the comment?

Reply:
done